.github/workflows/maas-api-release.yml

```diff
@@ -0,0 +1,38 @@
+name: MaaS API Release
+
+on:
+  push:
+    tags:
+      - 'v*'
+
+permissions:
```

Comment:
Related comment in the other PR: https://github.com/opendatahub-io/maas-billing/pull/107#discussion_r2388921932